webapi/src/main/scala/org/knora/webapi/Main.scala

```diff
@@ -23,7 +23,7 @@ object Main extends ZIOApp {
    * and cleaned up when the Runtime is shutdown.
    */
   override def bootstrap: ZLayer[
-    ZIOAppArgs with Scope,
```

Comment:
Basically it lgtm, I just wonder what motivates this change in `Main.scala`?

Reply:
this seems to have changed in the new ZIO version, presumably the Scope layer is now baked into it, so that it doesn't have to be explicitly provided.  
In any case, I didn't actually change any code, just the type annotation which wasn't correct anymore (and caused a compilation error). But if this hadn't been annotated explicitly, no change would have been needed.

Reply:
Make sense, thanks for the explanation.